### DISCUSSION TITLE
群消息发送失败: 账号可能被风控？

### DISCUSSION THREAD

**Discussion by feifeishenFFS** [Mrs4s/go-cqhttp#2081] [Q&A]
机器人发送长消息的时候，会显示这个报错。而相对较短的文本，则正常发送，请问该如何解决？

**Comment by VeryrrDefine** [comment]
如果发送消息内容很长或者被禁言可能会出现这个提示。
我建议你分条发送。

**Comment by axiangcoding** [comment]
同样的问题，测试了发现也不是字数的问题，似乎有什么奇怪的规则，只有短文本能发送
